### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mnatsheh**
Update dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,24 +31,25 @@
...
+    "@apollo/client": "^3.7.3",
     "@formatjs/intl-relativetimeformat": "^4.5.9",
     "@formatjs/intl-utils": "^2.2.0",
     "@manuscripts/assets": "^0.6.1",
-    "@manuscripts/eslint-config": "^0.4.2",
-    "@manuscripts/title-editor": "^0.5.2",
+    "@manuscripts/title-editor": "../manuscripts-title-editor",
+    "@manuscripts/comment-editor": "../manuscripts-comment-editor",
+    "@manuscripts/manuscript-transform": "../manuscripts-manuscript-transform",
+    "@manuscripts/manuscripts-json-schema": "../manuscripts-json-schema",
```
Comment (asouqi) [inline package.json:41]: dependencies should be with a version!

**Comment by mnatsheh** [comment]
Follow up work:
- Remove GraphQL and other LW-specific code.
- What are we using formatjs for? it's a huge library and it would be nice to get rid of it.
- Upgrade tooltip.js.

File: src/components/FileManager/FileManager.tsx [hunk 2]
```diff
@@ -162,20 +162,8 @@ export const FileManager: React.FC<{
   const handleUpdateInline = useCallback(
     async (modelId: string, attachment: SubmissionAttachment) => {
       const figureModel = modelMap.get(modelId) as Figure
-      const imageExternalFileIndex =
```
Comment (mbartenev-atypon) [inline src/components/FileManager/FileManager.tsx:165]: External file references removal is handled in LEAN-1620. It is a change across many components. Why is it handled here along with the dependencies update?

File: src/components/FileManager/InlineFilesSection.tsx [hunk 3]
```diff
@@ -37,12 +37,6 @@ import {
   fileTypesWithIconMap,
 } from './util'
 
-export interface ExternalFileRef {
```
Comment (mbartenev-atypon) [inline src/components/FileManager/InlineFilesSection.tsx:40]: Also in LEAN-1620

File: stories/data/article-doc.json [hunk 4]
```diff
@@ -136,145 +136,6 @@
             }
           ]
         },
-        {
-          "type": "multi_graphic_figure_element",
```
Comment (mbartenev-atypon) [inline stories/data/article-doc.json:140]: Why is this deleted? How is it related to dependencies update?

**Review by mbartenev-atypon — CHANGES_REQUESTED** [review]

File: src/components/FileManager/FileManager.tsx (hunk 2 above)
Comment (mnatsheh) [reply src/components/FileManager/FileManager.tsx:165]: I had to do this to get the code compile with the new transform/json-schema (which already has these things removed). Once your changes from LEAN-1620 are merged, I'll undo this change and rebase.

File: stories/data/article-doc.json (hunk 4 above)
Comment (mnatsheh) [reply stories/data/article-doc.json:140]: Part of the image changes was to remove the multi_graphic_figure_element node, so this doc fails to parse.

File: package.json (hunk 1 above)
Comment (mnatsheh) [reply package.json]: Check out my message in the group.

Comment (mbartenev-atypon) [reply package.json]: I'v read the message but it's not clear to me how is that going to work during deployments?

Comment (mbartenev-atypon) [reply package.json]: Or is that only for that specific change to be able to test things?

Comment (mnatsheh) [reply package.json]: This is only for local development, once these modules are released, this part will be updated with the new versions.
